packages/icons/src/metadata.json

```diff
@@ -0,0 +1,2608 @@
+{
+  "24h": {
+    "category": "Help & Support",
+    "aliases": ["24-hour", "around-the-clock", "24-Stunden", "rund-um-die-Uhr"]
```

Comment:
I don't think we can expect users to search for tthe hyphen-separated aliases, 
![image](https://github.com/user-attachments/assets/b73beccf-ef40-4e4d-8577-e3cfc300f053)
![image](https://github.com/user-attachments/assets/5bff13a4-247f-4239-abfa-31dedc4a68f5)
what do you think about extending our search functionality in http://localhost:5173/icons.html to also support space separated aliases? e.g. bei duplicating all aliases with something like `alias.split('-').join(' ')` 
maybe it might be more efficient to do this once when importing the file?

Reply:
can be extra PR!